app/javascript/src/Users/components/ServicesFieldset.jsx

```diff
@@ -0,0 +1,74 @@
+// @flow
+
+import React from 'react'
+
+import type { Service } from 'Types'
+import type { AdminSections } from 'Users/types'
+
+const ServicesFieldset = ({ services = [], selectedSections = [], selectedServicesIds = [], onServiceSelected }: {
+  services?: Service[],
+  selectedSections?: AdminSections[],
+  selectedServicesIds?: number[],
+  onServiceSelected: number => void
+}) => {
+  const servicesList = `ServiceAccessList`
```

Comment:
A problem that I see with this const name, is that I might be expecting an Array of services because of the "List" part. And this is just a css class name. I'd rather name it `serviceListClassName` or we could agree on something like `serviceListCss` or even all related css classes as `cssNameOfTheSet`. Also, use single quotes not template string. And to simplify it more, in this case, we might not even need a const to hold that sole value.

Reply:
I wasn't sure either, I wanted t avoid repetition but I agree it is misleading. I'll try with the ClassName at the end.